[2024-07-16 02:46:30,898] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-09-30T00:00:00+00:00 [queued]>
[2024-07-16 02:46:30,940] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-09-30T00:00:00+00:00 [queued]>
[2024-07-16 02:46:30,940] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:30,941] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-07-16 02:46:30,942] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:30,965] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-09-30T00:00:00+00:00
[2024-07-16 02:46:30,973] {standard_task_runner.py:52} INFO - Started process 3458 to run task
[2024-07-16 02:46:30,981] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2021-09-30T00:00:00+00:00', '--job-id', '949', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmphcxr4vhs', '--error-file', '/tmp/tmpa202su21']
[2024-07-16 02:46:30,988] {standard_task_runner.py:77} INFO - Job 949: Subtask second_task
[2024-07-16 02:46:31,139] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2021-09-30T00:00:00+00:00 [running]> on host f3060829e7b4
[2024-07-16 02:46:31,252] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-30T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-30T00:00:00+00:00
[2024-07-16 02:46:31,254] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-07-16 02:46:31,256] {bash.py:158} INFO - Running command: echo hello world, this is the second task, it will run after first task
[2024-07-16 02:46:31,270] {bash.py:169} INFO - Output:
[2024-07-16 02:46:31,273] {bash.py:173} INFO - hello world, this is the second task, it will run after first task
[2024-07-16 02:46:31,274] {bash.py:177} INFO - Command exited with return code 0
[2024-07-16 02:46:31,407] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20210930T000000, start_date=20240716T024630, end_date=20240716T024631
[2024-07-16 02:46:31,792] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-16 02:46:31,847] {local_task_job.py:146} INFO - Task exited with return code 0
